base_optional_quick_create/models/ir_model.py

```diff
@@ -26,6 +26,8 @@ def wrapper(self, name):
         method_name = 'name_create'
         for model in self:
             model_obj = self.env.get(model.model)
+            if not model_obj:
```

Comment:
I faced the same problem, and this approach did not work for me. Instead I suggest this:
```suggestion
            if not isinstance(model_obj, models.BaseModel):
```

Reply:
can you explain why you think this is necessary? For me the code fixes the problem as it is, and `get` is defined as returning something falsy if the requested key doesn't exist, so I'd be surprised if something else came back

Reply:
Yes, get will return something falsy it the key does not exist but also if the key **does** exist because the result will be an empty recordset.

Reply:
ah, thanks